static/css/style.css

```diff
@@ -6849,3 +6849,384 @@ ul.submit-options li:last-of-type {
 	}
 
 }
+
+.dei-hero-bg {
+	background: #0054C4;
+}
+
+.dei-hero-content {
+	max-width: 600px;
+	margin: 0 auto;
+
+}
+
+@media only screen and (max-width: 767px) {
+	.dei-hero-content {
+		width: 70%;
+	}
+}
+
+
+.dei-hero-heading {
+	font-family: Poppins;
+	font-size: 42px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 50px;
+	letter-spacing: 0px;
+	text-align: center;
+}
+
+.dei-hero-subheading {
+	font-family: Roboto;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 300;
+	line-height: 24px;
+	letter-spacing: 0px;
+	text-align: center;
+}
+
+.dei-hero-btn {
+	background: #fff;
+	color: #0055C6;
+}
+
+.dei-hero-btn:hover {
+	background: #0055C6;
+	color: #fff;
+}
+
+
+
+@media only screen and (max-width: 991px) {
+	.dei-at-storj-content {
+		display: flex;
+		flex-direction: column;
+	}
+	.dei-at-storj-text-wrapper {
+		order: 2;
+	}
+	.dei-at-storj-img-wrapper {
+		order: 1;
+		margin-bottom: 90px;
+	}
+}
+
+.dei-at-storj-heading {
+	font-family: Poppins;
+	font-size: 28px;
+	font-style: normal;
+	font-weight: 700;
+	line-height: 41px;
+	letter-spacing: 0px;
+	text-align: left;
+	margin-bottom: 0;
+}
+
+.dei-at-storj-subheading {
+	font-family: Roboto;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 300;
+	line-height: 24px;
+	letter-spacing: 0px;
+	text-align: left;
+}
+
+.dei-at-storj-btn {
+	float: left;
+}
+
+.dei-blm {
+	background: #000;
+	margin-bottom: 30px;
```

Comment:
Based on the design, the border around this content is on all sides, so replace with margin: 20px;